qiskit_ibm_runtime/options/estimator_options.py

```diff
@@ -12,101 +12,108 @@
+            collection of qubit-wise commuting observables for some specific circuit/parameter
+            value set to create a single measurement with basis rotations that is inserted into
+            hardware executions.
+
+            If twirling is enabled, the value of this option will be divided over circuit,
+            randomizations, with a smaller number of shots per randomization. See the
+            :attr:`~twirling` options.
+
+
         optimization_level: How much optimization to perform on the circuits.
             Higher levels generate more optimized circuits,
             at the expense of longer processing times.
+
             * 0: no optimization
             * 1: light optimization
 
+            Default: 1.
+
         resilience_level: How much resilience to build against errors.
             Higher levels generate more accurate results,
             at the expense of longer processing times.
 
             * 0: No mitigation.
             * 1: Minimal mitigation costs. Mitigate error associated with readout errors.
             * 2: Medium mitigation costs. Typically reduces bias in estimators but
-              is not guaranteed to be zero bias. Only applies to estimator.
+              is not guaranteed to be zero bias.
 
             Refer to the
             `Qiskit Runtime documentation
-            <https://qiskit.org/documentation/partners/qiskit_ibm_runtime>`_.
+            <https://docs.quantum.ibm.com/run/configure-error-mitigation>`_.
             for more information about the error mitigation methods used at each level.
 
-        dynamical_decoupling: Optional, specify a dynamical decoupling sequence to use.
-            Allowed values are ``"XX"``, ``"XpXm"``, ``"XY4"``.
-            Default: None
+            Default: 1.
 
         seed_estimator: Seed used to control sampling.
 
+        dynamical_decoupling: Suboptions for dynamical decoupling. See
+            :class:`DynamicalDecouplingOptions` for all available options.
+
         resilience: Advanced resilience options to fine tune the resilience strategy.
-            See :class:`ResilienceOptions` for all available options.
+            See :class:`ResilienceOptionsV2` for all available options.
 
         execution: Execution time options. See :class:`ExecutionOptionsV2` for all available options.
 
         twirling: Pauli twirling options. See :class:`TwirlingOptions` for all available options.
 
-        environment: Options related to the execution environment. See
-            :class:`EnvironmentOptions` for all available options.
-
-        simulator: Simulator options. See
-            :class:`SimulatorOptions` for all available options.
-
+        experimental: Experimental options.
     """
 
     # Sadly we cannot use pydantic's built in validation because it won't work on Unset.
+    default_precision: Union[UnsetType, float] = Unset
+    default_shots: Union[UnsetType, int] = Unset
```

Comment:
This one None is an allowed value, since it is only used to override default_precision (which always has to be float)
```suggestion
    default_shots: Union[UnsetType, int, None] = Unset
```

Reply:
How would `None` be different from `Unset`?